integration/tests/cook/test_basic.py

```diff
@@ -2838,3 +2838,68 @@ def test_cook_executor_reset_vars(self):
                 util.wait_for_instance(self.cook_url, uuid, status='success')
         finally:
             util.kill_jobs(self.cook_url, job_uuids, assert_response=False)
+
+
+    @unittest.skipUnless(util.has_docker_service() and util.docker_tests_enabled(), "Requires `docker inspect`")
+    def test_default_container_volumes(self):
```

Comment:
Can we write this test to have the command read and verify output of files mounted from the defaults and the user's mounted volumes to verify that mounts work as expected?